test/e2e/framework/sessionconfig.go

```diff
@@ -237,18 +238,29 @@ func (cfg SessionConfig) DeleteFARs() []*ie.IE {
 	}
 }
 
-func (cfg SessionConfig) SessionIEs() []*ie.IE {
+func (cfg SessionConfig) CreateURR (id uint32, triggers uint16) *ie.IE {
+	urr := ie.NewCreateURR(
+		ie.NewURRID(id),
+		ie.NewMeasurementMethod(0, 1, 1), // VOLUM=1 DURAT=1
+		ie.NewReportingTriggers(triggers))
+	// for Quota Validity Time report trigger is 15th bit in host order
+	if (triggers & 0x80) != 0 {
+		urr.Add(ie.NewQuotaValidityTime(time.Now()))
+
+		valTimer, _ := urr.FindByType(ie.QuotaValidityTime)
+		for i, _ := range valTimer.Payload {
+			valTimer.Payload[i] = 0
+		}
+		valTimer.Payload[3] = 10
+
+	}
+	return urr
+}
+
+func (cfg SessionConfig) SessionIEs(triggers uint16) []*ie.IE {
 	ies := cfg.CreateFARs()
 	if !cfg.NoURRs {
-		ies = append(ies,
-			ie.NewCreateURR(
-				ie.NewURRID(1),
-				ie.NewMeasurementMethod(0, 1, 1), // VOLUM=1 DURAT=1
-				ie.NewReportingTriggers(0)),
-			ie.NewCreateURR(
-				ie.NewURRID(2),
-				ie.NewMeasurementMethod(0, 1, 1), // VOLUM=1 DURAT=1
-				ie.NewReportingTriggers(0)))
+			ies = append(ies, cfg.CreateURR(1, triggers), cfg.CreateURR(2,triggers))
```

Comment:
If `triggers` is replaced by `ReportingTriggers` as suggested above, there will be no need to pass it here. Also, please don't forget about `go fmt`, I believe it can be automated (do it on save) on most editors

Reply:
Done